test/form/computed-properties/main.js

```diff
@@ -1,3 +1,12 @@
 var foo = 'foo';
+var bar = 'bar';
+var baz = 'baz';
+var bam = 'bam';
 
 export var x = {[foo]: 'bar'};
+
+export class X {
+  [bar]() {}
+  get [baz]() {}
+  set [bam](value) {}
```

Comment:
Code in the Rollup repo should be indented using tabs where possible. Good to merge once that's fixed.